### PR TITLE
Added new drawings to job notification emails

### PR DESCRIPTION
**Pull request by hotzenklotz**
PR adds new drawings to job notification emails in accordance with new branding.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Trigger a successful i.e. export as a tiff
- Upload a dataset to trigger a data uploaded email
- Start a segmentation to trigger a segementation finished email
- Stop the WK Worker and run a job to receive a job failed email

### Successful Upload
image

### Failed Job (Generic)
image

### Successful Job (Generic)
image

### Issues:
- Related to #6739 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: app/views/mail/jobSuccessfulGeneric.scala.html [hunk 1]
```diff
@@ -26,4 +26,11 @@
 </div>
 
 <p>With best regards,<br />the WEBKNOSSOS team</p>
+
+<div style="text-align: center;">
+  <img src="https://static.webknossos.org/mails/email-drawing-successful-job.png"
```
Comment (philippotto) [inline app/views/mail/jobSuccessfulGeneric.scala.html:31]: I get an access denied message for this url.

File: app/views/mail/jobSuccessfulUploadConvert.scala.html [hunk 2]
```diff
@@ -31,6 +31,17 @@
 <p>
   Did you know that we offer services for <a href="https://webknossos.org/services/alignment">dataset alignment & stitching</a> and <a href="https://webknossos.org/services/automated-segmentation">AI image segmentation</a> to kickstart your research?
 </p>
+<div style="text-align: center; margin-top: 10px; margin-bottom: 6px;">
```
Comment (philippotto) [inline app/views/mail/jobSuccessfulUploadConvert.scala.html:34]: unrelated to this pr, but in line 9 of this file, it says `amongst your other dataset` but it should be datasetS.

Comment (hotzenklotz) [reply app/views/mail/jobSuccessfulUploadConvert.scala.html:34]: Fixed

**Review by philippotto — APPROVED** [review]
lgtm :+1: I especially like the oops picture :sweat_smile:

**Pull request merged**